### ISSUE TITLE
Verify the list of urls

### ISSUE THREAD

**Issue by cybersiddhu** [dictyBase/frontpage-dictybase#75]
Evaluate and verify the list of url reserved by the app. Here is a simple checklist

* No special character like underscore and hypen.
* Everything needs to be lower case.
* Avoid more than two level of nesting.

Once it is verified and you made all the changes write the list here and i will reverify.

**Comment by biodavidjm** [comment]
The list of url currently used by the Angular app can be found in the `.config` function of the main script of the application [/app/scripts/app.js](https://github.com/dictyBase/frontpage-dictybase/blob/develop/app/scripts/app.js). These are:

```shell
/main
/about
/contact
/download
/citeus
/newsall
/papersall
/meetingsall
```

**Comment by biodavidjm** [comment]
Updated list of URLs (db9d80799f1efc97dc81eb52ffeff89d6b6a895b)

/main
/about
/download
/citeus
/news
/papers
/meetings

**Comment by cybersiddhu** [comment]
Here are recommendations

/main  =>  Broad meaning and ambigous, also could be noun or adjective.
/about  => Broad meaning and ambiguos, however it is usable in the context of dictybase.
/download => downloads
/citeus  => citation
/news  => OK
/papers => OK
/meetings => OK

**Comment by cybersiddhu** [comment]
I don't think you need `/papers` for frontpage application. You can use the url but the list of paper display should be handled by publictions/ontology application.  It is not the responsibility of frontpage app, it is preferred  to stick the what frontpage does now and not over extend it.
Instead, it is better to invest on websocket(better server side includes)  to refresh the list of new papers. The same implementation could be used for list of strains or anything that needs to be refreshed from time to time.
